pkg/blockstorage/blockstorage_test.go

```diff
@@ -272,6 +272,7 @@ func (s *BlockStorageProviderSuite) getConfig(c *C, region string) map[string]st
 		}
 		config[awsconfig.AccessKeyID] = accessKey
 		config[awsconfig.SecretAccessKey] = secretAccessKey
+		config[awsconfig.ConfigRole] = os.Getenv(awsconfig.ConfigRole)
```

Comment:
@tdmanv this enables the tests with  AWS assume role BUT it fails for me locally with the same error as we saw with before for storage_tests ```INFO[0000] Failed to create volume      error="UnauthorizedOperation: You are not authorized to perform this operation. Encoded authorization failure message```

Reply:
Tested successfully with different role.